packages/shared/routes/dashboard/wallet/Wallet.svelte

```diff
@@ -447,53 +444,51 @@
                 </div>
                 <div class="flex flex-1">
                     <DashboardPane classes="w-full rounded-tl-s rounded-tr-s">
-                        <AccountHistory
-                            transactions={getAccountMessages($selectedAccount)}
-                        />/>
+                        <AccountHistory transactions={getAccountMessages($selectedAccount)} />/>
```

Comment:
`/>/>` looks funky. I think we can remove one here

Reply:
haha I deleted that mora than three times at least, done thanks!